test/performance/README.md

```diff
@@ -1,128 +1,168 @@
-## Stress Test Experiments
+## KS Performance Experiments
 
 
 ## Generate the workload traffic
 
-1. Clone the following fork of the clusterloader2 repo: 
+1. Clone the following clusterloader2 repo: 
 
-```bash
-$ git clone -b ks-provider https://github.com/dumb0002/perf-tests.git
-```
+   ```bash
+   $ git clone -b release-1.31 https://github.com/kubernetes/perf-tests.git
+   ```
 
-2. Configure the parameters of your experiment:  
+   Set the variable `WORK_DIR` to the path of the directory where you cloned your repo:
+
+   ```bash
+   $ export WORK_DIR=$HOME/perf-test/clusterloader2
+   ```
+
+2. Configure clusterloader2 to generate KS performance workloads:
+
+   Starting from a local directory containing this KubeStellar git repo, run the following script:
+
+   For K8s clusters:
+
+   ```bash
+   $ ./setup-clusterloader2.sh
+   ```
+
+   In Kubernetes environments, a modified version of the kube-burner [cluster-density](https://github.com/kube-burner/kube-burner/tree/main/examples/workloads/cluster-density) workload will be generated per namespace. This workload consists of the following objects:
+
+   - 1 deployments, with two pod replicas (pause), mounting 2 secrets, 2 config maps
+   - 3 services, the first service points to the TCP/8080 port of the deployments, respectively.
+   - 10 secrets containing a 2048-character random string.
+   - 10 config maps
+
+   For Openshit clusters: 
+
+   ```bash
+   $ ./setup-clusterloader2.sh --env ocp
+   ```
+
+    In Openshift environments, a modified version of the kube-burner [cluster-density-ms](https://github.com/kube-burner/kube-burner-ocp/tree/478bb42e1842a94ca3210d26a08633b70a443005/cmd/config/cluster-density-ms) workload will be generated per namespace. This workload consists of the following objects:
+
+    - 1 image stream.
+    - 4 deployments, each with two pod replicas (pause), mounting 4 secrets, 4 config maps, and
+      1 downward API volume each.
+    - 2 services, each pointing to the TCP/8080 and TCP/8443 ports of the first and second
```

Comment:
This seems more different from the plain Kubernetes case than necessary?

Reply:
This is the exact list of objects from the sample workload extract from kube-burner for both k8s and ocp envs

Reply:
Oh, this difference comes from https://github.com/kube-burner/kube-burner/tree/main/examples/workloads/cluster-density vs https://github.com/kube-burner/kube-burner-ocp/tree/main/cmd/config/cluster-density-ms . Why do those two differ in number of Deployments, Services, and Secrets?

Reply:
Great question! I don't know the answer.